gae/gae.go

```diff
@@ -47,15 +47,20 @@ func NewMartini(option MartiniOption) *ExMartini {
 				continue
 			}
 
-			id := uuid.NewUUID()
+			id, err := uuid.NewRandom()
+			if err != nil {
+				aelog.Warningf(ac, "%s", err.Error())
+				continue
+			}
+
 			uuidString := id.String()
 			now := time.Now()
 			record := make(map[string]interface{})
 			record["id"] = uuidString
 			record["date"] = now
 			record["log"] = l
 
-			err := bq.SendLog(ac, "debuglog", uuidString, record)
+			err = bq.SendLog(ac, "debuglog", uuidString, record)
```

Comment:
Sorry, this is not PR comment but task request.
I've totally forgot this SendLog feature.
Please check if it's used in the caterpillar and other sub projects.